int32[] dxl_id
int32[] present_temp
int32[] present_load
int32[] present_volt
int32[] present_current
int32[] present_pos
int32[] present_vel
int32[] goal_pos
int32[] goal_vel
int32[] return_delay_time
int32[] feedforward_1st_gain
int32[] feedforward_2nd_gain
int32[] error_status
int32[] temp_limit
int32[] pos_p_gain
int32[] pos_i_gain
int32[] pos_d_gain
int32[] vel_p_gain
int32[] vel_i_gain
